charts/cluster-scanner/tests/deployment_test.yaml

```diff
@@ -92,9 +92,9 @@ tests:
     asserts:
       - not: true
         isEmpty:
-          path: spec.template.spec.containers[0].env[9]
+          path: spec.template.spec.containers[0].env[12]
```

Comment:
Please use a better way to identify the env node, eg:
https://github.com/sysdiglabs/charts/blob/master/charts/agent/tests/conditional_flag_test.yaml#L17
(same for the other nodes here)